nautilus_core/model/src/python/types/balance.rs

```diff
@@ -66,7 +66,7 @@ impl AccountBalance {
     }
 
     #[pyo3(name = "to_dict")]
-    fn py_to_dict(&self, py: Python<'_>) -> PyResult<PyObject> {
+    pub fn py_to_dict(&self, py: Python<'_>) -> PyResult<PyObject> {
```

Comment:
I think the `pub` access modifier will only affect Rust, anything in `pymethods` will be exposed to Python.